File: app/src/main/java/com/project/scratchstudio/kith_andoid/SetInternalData/SetCountData.java
package com.project.scratchstudio.kith_andoid.SetInternalData;

import android.content.SharedPreferences;

import com.project.scratchstudio.kith_andoid.Model.Cache;

public class SetCountData implements ISetInternalData {
    @Override
    public int set(SharedPreferences sp) {
        SharedPreferences.Editor ed;
        ed = sp.edit();
        ed.putInt("count_users", Cache.getAllUsers());
        ed.apply();
        return 0;
    }
}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/SetInternalData/SetCountData.kt
package com.project.scratchstudio.kith_andoid.SetInternalData

import android.content.SharedPreferences

import com.project.scratchstudio.kith_andoid.model.Cache

class SetCountData : ISetInternalData {
    override fun set(sp: SharedPreferences): Int {
        val ed: SharedPreferences.Editor = sp.edit()
        ed.putInt("count_users", Cache.allUsers)
        ed.apply()
        return 0
    }
}
